lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -290,11 +278,14 @@ class Object < ObjectAttribute
     end
 
     class Operation < PolicyElement
-      has_and_belongs_to_many :policy_element_associations, class_name: 'PolicyMachineStorageAdapter::ActiveRecord::PolicyElementAssociation', join_table: 'operations_policy_element_associations'
```

Comment:
Nope, I was thinking of this call. If we try to get this list of operation sets for the `read` operation, for example, then this could load hundreds of thousands of operation sets. The raw SQL (filter_operation_set_list_by_assigned_operation) in the psql adapter is our way around using this call.